nodes/standalone/src/chain_spec.rs

```diff
@@ -162,45 +121,47 @@ impl Alternative {
 					None,
 				)
 			}
-			Alternative::MashnetStaging => {
+			Alternative::SporranTestnet => {
+				properties.insert("tokenSymbol".into(), "SILT".into());
 				ChainSpec::from_genesis(
-					"Mashnet Staging",
-					"mashnet_staging",
-					ChainType::Live,
+					"Sporran",
+					"sporran",
+					ChainType::Development,
 					move || {
 						testnet_genesis(
 							wasm_binary,
-							// Initial Authorities
-							vec![
-								as_authority_key(DEV_AUTH_ALICE),
-								as_authority_key(DEV_AUTH_BOB),
-								as_authority_key(DEV_AUTH_CHARLIE),
-							],
-							DEV_AUTH_ALICE.into(),
-							vec![
-								DEV_FAUCET.into(),
-								DEV_AUTH_ALICE.into(),
-								DEV_AUTH_BOB.into(),
-								DEV_AUTH_CHARLIE.into(),
-							],
+							vec![(
+								SPORRAN_AUTHORITY_ACC.into(),
+								SPORRAN_AUTHORITY_SESSION.unchecked_into(),
+								SPORRAN_AUTHORITY_SESSION.unchecked_into(),
+							)],
+							SPORRAN_AUTHORITY_ACC.into(),
+							vec![SPORRAN_FAUCET.into(), SPORRAN_AUTHORITY_ACC.into()],
 						)
 					},
-					vec![],
-					None,
-					None,
+					vec![
+						"/dns4/bootnode.kilt.io/tcp/30340/p2p/12D3KooWGXaTjB6KmPHxyCx2dQLpS7p9vnXAYmprQMsVNnxYAWYa"
+							.parse()
+							.expect("bootnode address is formatted correctly; qed"),
+					],
+					Some(
+						TelemetryEndpoints::new(vec![(TELEMETRY_URL.to_string(), 0)])
+							.expect("WILT telemetry url is valid; qed"),
+					),
+					Some("SILT"),
 					Some(properties),
 					None,
 				)
 			}
+			Alternative::KiltTestnet => ChainSpec::from_json_bytes(&include_bytes!("../res/testnet.json")[..])?,
 		})
 	}
 
 	pub(crate) fn from(s: &str) -> Option<Self> {
 		match s {
 			"dev" => Some(Alternative::Development),
 			"kilt-testnet" => Some(Alternative::KiltTestnet),
-			"kilt-devnet" => Some(Alternative::KiltDevnet),
-			"mashnet-staging" => Some(Alternative::MashnetStaging),
+			"sporran-new" => Some(Alternative::SporranTestnet),
```

Comment:
C: Should this maybe be rather something like `sporran-testnet`? Since the word `new` might not tell anyone much looking at it in the future.

Reply:
It's called `sparran-new` since it creates a new chainspec. For running a node you always need to use the file `--chain dev-specs/mashnet-node/sporran.json`.

Reply:
ah ok, understood. Than fine for me